src/lib.rs

```diff
@@ -19,7 +19,15 @@
 //! base64-encoded DER, PEM objects are delimited by header and footer lines which indicate the type
 //! of object contained in the PEM blob.
 //!
-//! The [rustls-pemfile](https://docs.rs/rustls-pemfile) crate can be used to parse PEM files.
+//! Types here can be created from:
+//!
+//! - DER using (for example) [`PrivatePkcs8KeyDer::from`].
+//! - PEM using (for example) [`PrivatePkcs8KeyDer::from_pem_slice`].
+//!
+//! `from_pem_slice` returns the first matching item from the given input.
+//! `iter_pem_slice` returns every matching item from the given input.
+//! It is usual for a given PEM file to contain multiple items of some types,
```

Comment:
Nit: suffix function/method calls with `()`?

(I guess most of these are addressed in your docs fixup, but maybe not all of them.)

Reply:
addressed this remaining one